README.md

```diff
@@ -80,22 +80,21 @@ If you have no need for the UI assets git-lfs is optional.
 
 
 # Component list
-- [`ckcore`](ckcore/) the platform maintaining the [MultiDiGraph](https://en.wikipedia.org/wiki/Multigraph#Directed_multigraph_(edges_with_own_identity)).
-- [`cksh`](cksh/) the Cloudkeeper shell to interact with the core.
-- [`ckui`](ckui/) a UI prototype that can load ckcore exported data but has no backend connection yet.
-- [`ckworker`](ckworker/) provides workers that load [plugins](plugins/) to perform collect and cleanup operations.
-- [`ckmetrics`](ckmetrics/) is a [Prometheus](https://prometheus.io/) [exporter](https://prometheus.io/docs/instrumenting/exporters/).
+- [`resotocore`](resotocore/) the platform maintaining the [MultiDiGraph](https://en.wikipedia.org/wiki/Multigraph#Directed_multigraph_(edges_with_own_identity)).
+- [`resotosh`](resotosh/) the Resoto shell to interact with the core.
+- [`ui`](ui/) a UI prototype that can load resotocore exported data but has no backend connection yet.
+- [`resotoworker`](resotoworker/) provides workers that load [plugins](plugins/) to perform collect and cleanup operations.
+- [`resotometrics`](resotometrics/) is a [Prometheus](https://prometheus.io/) [exporter](https://prometheus.io/docs/instrumenting/exporters/).
 - [`plugins`](plugins/) are a collection of worker plugins like [AWS](plugins/aws/)
 
 
 ## Contact
-If you have any questions feel free to [join our Discord](https://discord.gg/someengineering) or [open a GitHub issue](https://github.com/someengineering/cloudkeeper/issues/new).
+If you have any questions feel free to [join our Discord](https://discord.gg/someengineering) or [open a GitHub issue](https://github.com/someengineering/resoto/issues/new).
 
 
 ## License
 ```
-Copyright 2021 Some Engineering Inc.
-Copyright 2019-2021 D2iQ, Inc.
+Copyright 2022 Some Engineering Inc.
```

Comment:
Isn't a range typically used in this case?  E.g., `2021-2022`?

Reply:
No, I read up on it. The only relevance the date has is the most recent date for which the copyright applies. So people would know when the copyright expires.